test/shared/ThisContext-component.test.js

```diff
@@ -15,14 +15,14 @@ class Foo extends React.Component {
 
 describe('ThisContext: load', () => {
   let self;
-  before(() => {
-    bdd.reset();
-    bdd.register();
-  });
 
+  let suite;
+  afterEach(() => { bdd.reset(); });
   beforeEach(() => {
-    api.reset({ hard: true });
-    self = new ThisContext();
+    bdd.register();
+    suite = describe(`My Suite`, () => {});
+    api.setCurrent({ suite });
```

Comment:
I'm not sure if this line is needed, when `this.component` is called, it should do this itself... @philcockfield can you give some more insight>